### PR TITLE
[Fixes #346] Improved designs and added a student's project

### LINKED ISSUE #346: Add student's project and improve the project page
**Actual Behaviour**
There is only 4 projects.

**Expected Behaviour**
Add another project and improve the design.

**Steps to reproduce it**

Add a student's project and improve the design.

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by sahilsaha7773**
- [x] Included a Preview link and screenshot showing after and before the changes.
- [x] Included a description of the change below.

# Changes done in this Pull Request
Fixes #346 

- [Preview Link](https://sahilsaha7773.github.io/projects.html)

## Description / Changes
I have improved the designs and added my project

### CODE REVIEW

**Comment by sahilsaha7773** [comment]
I have done it as a GCI task but I don't find any specific issues for these

**Comment by liveHarshit** [comment]
You have to create an issue if not already there, also you can comment both PR link in the task comment after the PR get merged.

**Comment by sahilsaha7773** [comment]
@liveHarshit  ok I am doing it

**Comment by sahilsaha7773** [comment]
Now I have an issue and this PR is solving that issue only.

**Comment by sahilsaha7773** [comment]
please review it now

File: students_projects.html [hunk 1]
```diff
@@ -1,7 +1,18 @@
 ---
 layout: default-pages
 ---
+<style>
+    #back {
+      background-color: #ea1010;
+
```
Comment (sridharjajoo) [inline students_projects.html]: Remove this extra line

Comment (sahilsaha7773) [reply students_projects.html]: ok

Comment (sahilsaha7773) [reply students_projects.html]: done

**Comment by sahilsaha7773** [comment]
Please review it now

**Comment by sahilsaha7773** [comment]
I am new in git so can you please help me in squashing the commits

**Comment by sridharjajoo** [comment]
Follow this [link](https://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git)

**Comment by sahilsaha7773** [comment]
Thanks but I think I have solved one issue only. @sarnava1

**Comment by sahilsaha7773** [comment]
Please check

File: students_projects.html [hunk 2]
```diff
@@ -1,7 +1,17 @@
 ---
 layout: default-pages
 ---
+<style>
```
Comment (fragm3) [inline students_projects.html]: Use class, and style it in separate css file(instead of inline styling).

Comment (sahilsaha7773) [reply students_projects.html]: will inline styling create any problem?

Comment (sahilsaha7773) [reply students_projects.html]: I have done internal styling here

**Comment by sahilsaha7773** [comment]
Please merge if it looks correct

File: students_projects.html (hunk 2 above)
Comment (sk9331657) [reply students_projects.html]: @sahilsaha7773 Adding inline css breaks uniformity . @fragm3  is right. Use external css.

**Comment by fragm3** [comment]
Closing this, duplicate of https://github.com/fossasia/gci18.fossasia.org/pull/359

**Pull request closed without merging**